bioio/tests/test_bio_image.py

```diff
@@ -74,3 +84,32 @@ def test_bioimage_submission_data_reader_type_alignment(
 ) -> None:
     with pytest.raises(expected_exception):
         BioImage(image, reader=reader_class)
+
+
+def test_dump_plugins() -> None:
+    package_name = "dummy-plugin"
+
+    try:
+        # Install the plugin
+        subprocess.check_call([sys.executable, "-m", "pip", "install", DUMMY_PLUGIN])
```

Comment:
I love the mock plugin, just wish there were a simpler way to "install" it.  IDK this is fine tho.